api.planx.uk/modules/analytics/metabase/collection/service.ts

```diff
@@ -0,0 +1,33 @@
+import { updateMetabaseId } from "./updateMetabaseId.js";
+import type { NewCollectionParams } from "./types.js";
+import { getTeamIdAndMetabaseId } from "./getTeamIdAndMetabaseId.js";
+import { createCollection } from "./createCollection.js";
+
+/**
+ * First uses name to get teams.id and .metabase_id, if present.
+ * If metabase_id is null, return an object that includes its id. If not, create the collection.
+ * @params `name` is required, but `description` and `parent_id` are optional.
+ * @returns `response.data`, so use dot notation to access `id` or `parent_id`.
+ */
```

Comment:
That makes sense, I think it was inexperience that led me to writing some 'what' comments for myself :) 

In this case I think there was _some_ slightly better reasoning too: that it was hard to come up with a name that also communicated the `try ... getTeamIdAndMetabaseId()` functionality. I've tried to update the comment (28712c4f) to explain a bit more of the why rather than the what, let me know what you think!